integration-tests/webhook-perf-test/pom.xml

```diff
@@ -0,0 +1,198 @@
+<?xml version="1.0"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.redhat.service.smartevents</groupId>
```

Comment:
Maybe we should make this artifact dependent of integration-tests pom, so we the deps can be managed at that level. But I'd say this can be done later in a different PR.

Reply:
Can be done separately, my initial experiments showed that there is some dependency incompatibility which would have to be addressed.